src/box/lua/load_cfg.lua

```diff
@@ -447,7 +447,7 @@ local dynamic_cfg_skip_at_load = {
     too_long_threshold      = true,
     election_mode           = true,
     election_timeout        = true,
-    election_fencing_enabled = true,
+    election_fencing_mode   = true,
```

Comment:
Please add `election_fencing_enabled` to `translate_cfg` table. The option was released already, and it can't simply disappear.
I suppose "fencing_enabled = true" should translate to "fencing_mode='soft'"

Reply:
Thanks for noticing, didn't know we had such functionality. Fixed